### PR TITLE
Update carto-package.json

### PR DESCRIPTION
**Pull request by lekum**
Versions fixed

### CODE REVIEW

File: carto-package.json [hunk 1]
```diff
@@ -2,15 +2,15 @@
...
             "npm": ">=3.10.9",
             "gdal":">=1.11.0"
         },
         "works_with": {
             "redis": ">=3.0.0",
-            "postgresql": ">=9.5",
-            "postgis": ">=2.2",
-            "carto_postgresql_ext": ">=0.19"
+            "postgresql": ">=9.5.0",
+            "postgis": ">=2.2.0.0",
```
Comment (oleurud) [inline carto-package.json]: do you support 4 digists?

Comment (lekum) [reply carto-package.json]: Yes, we support them using a "hack" (changing `a.b.c.d` to `a.b.c-d`) but only for components that we cannot easily adapt the tagging (as `postgis`)

**Comment by lekum** [comment]
May I merge it then?

**Comment by oleurud** [comment]
Yes, you have my blessings!

**Pull request merged**